### ISSUE TITLE
Background and Alignment of footer section

### ISSUE THREAD

**Issue by madhhuurrii** [garimasingh128/Hacktoberfest2020-MovieDo#1]
**Describe the bug**
Background is not present in footer section and unalignment of footer content i.e it is in extreme left.

**To Reproduce**
1. Go to live website 
2. Scroll down to footer section
3. See error

**Expected behavior**
Background should be according to description in readme file i.e dark blue.
Alignment of content should be at center

**Screenshots**
![Screenshot_20200925-050416_Samsung Internet](https://user-images.githubusercontent.com/64744084/94211001-f0dde680-feed-11ea-833a-691ebc160161.jpg)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy M30s
 - OS: Android 10.0 (Q) with One UI 2.0
 - Browser : Samsung Internet
 - Version : Android 10

**Additional context**

**Comment by SoldierCorp** [comment]
Hi,

I would like to help, can you assign it to me please?

**Comment by garimasingh128** [comment]
Already PR #6 is created for it. You may create more issues or simply add your fav movie to data.json file.

**Comment by madhhuurrii** [comment]
Hii @garimasingh128 , actually wanted to know is the PR made before oct 1 are counted for hacktoberfest cos i was willing to work on this issue but was actually waiting for the dates

**Comment by garimasingh128** [comment]
Hacktoberfest 2020 has begun as seen on the website. Go and register there asap. I think the PR would be counted but you can wait until 1st if you want.

**Comment by devcshort** [comment]
@garimasingh128 @madhhuurrii this is incorrect. Hacktoberfest has not started yet and any PRs created before October 1st will not count.

In their FAQs it states

> all pull requests created between Oct 1 and Oct 31 will count, regardless of when you register for Hacktoberfest. Pull requests created before Oct 1 but merged or marked as ready for review after do not count.

You can see [here](https://hacktoberfest.digitalocean.com/faq/) under the **Rules** section

**Comment by garimasingh128** [comment]
My apologies for the same. @madhhuurrii you may close your PR and open another one after 1st Oct.

**Comment by SoldierCorp** [comment]
Hi @garimasingh128,

People are looking for unassigned issues and this issue was already covered by @madhhuurrii but it hasn't been assigned yet and is still appearing in the search results so you should assign it to @madhhuurrii please

**Comment by garimasingh128** [comment]
It would be fine right if I assign multiple people to the same issue?

**Comment by madhhuurrii** [comment]
Hii @garimasingh128 , I have solved the issue , please review it

**Comment by madhhuurrii** [comment]
Hii @garimasingh128 would like to know if this repository's contribution count towards hacktoberfest 2020.
